### DISCUSSION TITLE
Does wix / Heatwave support Indonesian installer?

### DISCUSSION THREAD

**Discussion by disco-sai-gattu** [wixtoolset/issues#7701] [Questions] [answered]
## Triage

Issues are triaged at [online meetings](https://www.firegiant.com/blog/), generally held alternate Tuesdays at 9:30 a.m. Pacific time (UTC-7/UTC-8). Meeting requests are posted in [Discussions](https://github.com/orgs/wixtoolset/discussions/categories/announcements).

## Discussions and support

Note that discussions are best held in [Discussions](https://github.com/orgs/wixtoolset/discussions/new?category=questions). Please do not open issues requesting support or debugging help; first start a thread in [Discussions](https://github.com/orgs/wixtoolset/discussions/new?category=questions) and open an issue if the discussion suggests a bug in the WiX Toolset or that a feature request is appropriate.

## Instructions

Read the sections above. Delete the Triage, Discussions and support, and Instructions sections and fill out the rest.

## Bugs

If this issue is a bug:

* Which version of WiX are you building with?

> (WiX version in major.minor.release.build format)

* Which version of Visual Studio are you building with (if any)?

> (Visual Studio version, including year and update)

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

> (WiX Toolset Visual Studio Extension version in major.minor.release.build format)

* Which version of .NET are you building with?

> (.NET version)

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> (Windows version)

* Describe the problem and the steps to reproduce it.

> (Be as specific as you can and err on the side of providing too much information, including code, error messages, command lines you used to invoke the build, and so forth.)

* Describe the behavior you expected and how it differed from the actual behavior.

> (Be as specific as you can and err on the side of providing too much information.)

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [GitHub discussions area][gh]. You might also consider asking your question on [Stack Overflow][so].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions
[so]: https://stackoverflow.com/questions/tagged/wix

**Comment by BMurri** [comment, marked as answer]
Depends on your definition of "support". MSI's `SummaryInformation` does not support unicode, so you will need to use a non-unicode codepage supported by computers with the `bahasa Indonesia` language pack installed that covers all of the characters that you might place into the fields that are included in any summary info fields. You can use `utf-8` for all other codepages, ensuring support for any character the computer is capable of displaying. The rest of "wix/heatwave" use unicode everywhere.

You'll probably want to use the LCID of `1057` if I'm not mistaken for the `Language` value (not `33`) and `id-ID` for the `Culture` value.

See https://wixtoolset.org/docs/tools/codepage/ for more information.
